python/pyspark/sql/pandas/serializers.py

```diff
@@ -371,3 +375,292 @@ def load_stream(self, stream):
                 raise ValueError(
                     "Invalid number of pandas.DataFrames in group {0}".format(dataframes_in_group)
                 )
+
+
+class ApplyInPandasWithStateSerializer(ArrowStreamPandasUDFSerializer):
```

Comment:
can we have some class and method level comments here.

Reply:
Done.